### PR TITLE
Regenerate and fix AMD and EXT extension html files

### PR DESCRIPTION
**Pull request by Tobski**
This regenerates the html files for AMD and EXT extensions using the current toolchain, and fixes several errors with table rendering.

### CODE REVIEW

**Review by dgkoch — COMMENTED** [review]
I scanned through all the HTML before vs after. 

Only spotted issues in the AMD extensions, which you might want to fix, but you can choose to ignore if you want since they are your extensions.

No issues spotted in the EXT extensions.

File: extensions/AMD/SPV_AMD_gcn_shader.html
[the remark this answers is not in the document]
Comment (Tobski) [reply extensions/AMD/SPV_AMD_gcn_shader.html:1]: How on earth did we manage that. Ok I'll see if I can update the source to match the currently published spec. Thanks.

**Comment by Tobski** [comment]
@dgkoch thanks for the review, all of the issues you've pointed out should be fixed.

File: extensions/AMD/SPV_AMD_gcn_shader.asciidoc [hunk 1]
```diff
@@ -175,4 +170,5 @@ Revision History
 |========================================
 |Rev|Date|Author|Changes
 |1|May 31, 2016|Dominik Witczak|Initial revision based on AMD_gcn_shader.
+|1|October 13, 2016|Dominik Witczak|Added missing numerical value assignments, removed extension numbe
```
Comment (dgkoch) [inline extensions/AMD/SPV_AMD_gcn_shader.asciidoc:173]:
```suggestion
|1|October 13, 2016|Dominik Witczak|Added missing numerical value assignments, removed extension number
```

**Comment by bashbaug** [comment]
Merging as discussed in the October 23rd teleconference.

**Pull request merged**